cli/init.md

```diff
@@ -81,84 +86,98 @@ What happens behind the scenes?
+- The `#current-cloud-backend` directory has a similar structure to the `backend` directory. The only differences between it and the `backend` directory are that it has the configurations that reflect what resources were deployed in the cloud with your last `amplify push` command and it helps the CLI diff between the configuration of the resources already provisioned in the cloud and what is currently in your local `backend` directory (which reflects your local changes).
 
-- The `.config` directory consists of the metadata files tied to your project. The `project-config.json` file has the following format:
+- The `.config` directory consists of the metadata files tied to your project. 
+The `project-config.json` file (this file could be safely checked into a version control system) has the following format which represents information specific to the app you're building, for example the framework you're using for your app:
 
 ```
 {
-    "projectName": "test890",
-    "projectPath": "/Users/kaustavg/test890",
-    "defaultEditor": "sublime",
-    "frontendHandler": {
-        "javascript": "/Users/xxxxx/aws-amplify-cli/amplify-cli/packages/amplify-cli/node_modules/amplify-frontend-javascript"
-    },
+    "projectName": "testapp",
     "javascript": {
-        "framework": "none",
+        "framework": "react",
         "config": {
             "SourceDir": "src",
             "DistributionDir": "dist",
             "BuildCommand": "npm run-script build",
             "StartCommand": "npm run-script start"
         }
     },
-    "providers": {
-        "awscloudformation": "/Users/xxxxx/aws-amplify-cli/amplify-cli/packages/amplify-cli/node_modules/amplify-provider-awscloudformation"
-    }
+    "providers": [
+        "awscloudformation"
+    ],
+    "frontend": "javascript",
+    "version": "1.0"
 }
 ```
 
-Note: The `project-config.json` file has your local system-specific paths and shouldn't be checked in to your source control if sharing the project with another team member working on another system.
 
-The `.config` directory also has a `aws-info.json` file which lets the CLI know which AWS profile/accesskey-secret key pair to use when adding AWS resources to your project. If you're using an AWS profile to initialize your project, the format should be the following:
+The `.config` directory also has a `local-aws-info.json` file (this file should not be checked into version control since it has information specific to a system on which the CLI is running on) which lets the CLI know which AWS profile/accesskey-secret key pair to use when adding AWS resources to your project. If you're using an AWS profile to initialize your project, the format should be the following: :
```

Comment:
The `.config` directory also has a `local-aws-info.json` file that lets the CLI know which AWS profile/accesskey-secret key pair to use when adding AWS resources to your project. 
_Note: This file should not be checked into version control since it has information specific to a system on which the CLI is running on._  
If you're using an AWS profile to initialize your project, the format should ........

Reply:
Are we automatically adding this to .gitignore?